wtsi-team112/ncov2019-artic-nf
# tests that run on master pull requests

tests:
- build singularity images
- run pipeline and compare output with upstream fork (Connor lab)
- run Sanger profile and check it ran `bwa` with 4 cpus and `ivar` without `-e`
